experimental/packages/opentelemetry-instrumentation/README.md

```diff
@@ -219,6 +219,13 @@ If nothing is specified the global registered provider is used. Usually this is
 There might be usecase where someone has the need for more providers within an application. Please note that special care must be takes in such setups
 to avoid leaking information from one provider to the other because there are a lot places where e.g. the global `ContextManager` or `Propagator` is used.
 
+## Instrumentation for ES Modules In NodeJS (experimental)
+
+As the module loading mechanism for ESM is different than CJS, you need to select a custom loader so instrumentation can load hook on the esm module it want to patch. To do so, you must provide the `--experimental-loader=@opentelemetry/instrumentation/hook.mjs` flag to the `node` binary. Alternatively you can set the `NODE_OPTIONS` environment variable to `NODE_OPTIONS="--experimental-loader=@opentelemetry/instrumentation/hook.mjs"`.
+As the ESM module loader from NodeJS is experimental, so is our support for it. Feel free to provide feedback or report issues about it.
+
+**Note**: ESM Instrumentation is not yet supported for Node 20.
+
```

Comment:
Please update the limitations section below which tells `ECMA script modules (using `import`) is not supported as of now`.

Reply:
what does this mean? How can one use ESM without import?

Reply:
It means that the limitations section requires changes in this PR because it removes one of them.
Before this PR ESM/`import` was not supported and only CJS/`require` worked.
Th limitation with bundlers stays.

Reply:
huh, super, thanks for the clarification :) a panic mode was already about to arise :D